doc/persistent_scenes_and_states/move_group.launch.py

```diff
@@ -0,0 +1,230 @@
+    )
+    ## END_SUB_TUTORIAL
+
+    # RViz
+    rviz_config_file = (
+        get_package_share_directory("moveit2_tutorials") + "/launch/move_group.rviz"
+    )
+
+    ## BEGIN_SUB_TUTORIAL set_config_rviz
+    ## * and to the RViz config
+    rviz_node = Node(
+        package="rviz2",
+        executable="rviz2",
+        name="rviz2",
+        output="log",
+        arguments=["-d", rviz_config_file],
+        parameters=[
+            robot_description,
+            robot_description_semantic,
+            ompl_planning_pipeline_config,
+            kinematics_yaml,
+            # here
+            warehouse_ros_config,
+        ],
+    )
+    ## END_SUB_TUTORIAL
+
+    # Static TF
+    static_tf = Node(
+        package="tf2_ros",
+        executable="static_transform_publisher",
+        name="static_transform_publisher",
+        output="log",
+        arguments=["0.0", "0.0", "0.0", "0.0", "0.0", "0.0", "world", "panda_link0"],
+    )
+
+    # Publish TF
+    robot_state_publisher = Node(
+        package="robot_state_publisher",
+        executable="robot_state_publisher",
+        name="robot_state_publisher",
+        output="both",
+        parameters=[robot_description],
+    )
+
+    # ros2_control using FakeSystem as hardware
+    ros2_controllers_path = os.path.join(
+        get_package_share_directory("moveit_resources_panda_moveit_config"),
+        "config",
+        "panda_ros_controllers.yaml",
+    )
+    ros2_control_node = Node(
+        package="controller_manager",
+        executable="ros2_control_node",
+        parameters=[robot_description, ros2_controllers_path],
+        output={
+            "stdout": "screen",
+            "stderr": "screen",
+        },
+    )
+
+    # Load controllers
+    load_controllers = []
+    for controller in [
+        "panda_arm_controller",
+        "panda_hand_controller",
+        "joint_state_controller",
+    ]:
+        load_controllers += [
+            ExecuteProcess(
+                cmd=["ros2 run controller_manager spawner.py {}".format(controller)],
+                shell=True,
+                output="screen",
+            )
+        ]
+
+    # Warehouse mongodb server
+    ## BEGIN_SUB_TUTORIAL start_mongodb_server
+    ## * start the MongoDB server (if necessary)
+    mongodb_server_node = Node(
+        package="warehouse_ros_mongo",
+        executable="mongo_wrapper_ros.py",
+        parameters=[
+            warehouse_ros_config,
+        ],
+        output="screen",
+    )
+    ## END_SUB_TUTORIAL
+
+    return LaunchDescription(
+        [
+            rviz_node,
+            static_tf,
+            robot_state_publisher,
+            run_move_group_node,
+            ros2_control_node,
+            mongodb_server_node,
+        ]
+        + load_controllers
+    )
+
+
+## BEGIN_TUTORIAL
+## CALL_SUB_TUTORIAL add_config
+## CALL_SUB_TUTORIAL set_config_move_group
+## CALL_SUB_TUTORIAL set_config_rviz
+## CALL_SUB_TUTORIAL start_mongodb_server
+## END_TUTORIAL
```

Comment:
I'm not familiar with the current MoveIt launch file code in ROS2 (In my naive opinion it's a horrible and entirely useless pile of crap that does not scale), so I can't review this file at all.

Please review this file @JafarAbdi @henningkayser 
Do you really add these whole stacks to each and every tutorial?